src/AppBundle/Resources/sass/blocks/_block.scss

```diff
@@ -18,6 +18,38 @@
               padding-bottom: $valueMap;
         }
     }
+
+    @include media-breakpoint-down(sm) {
+        @each $keyMap, $valueMap in $sm-sizes {
+            &.whitespace-top-#{$keyMap} {
+                  padding-top: $valueMap;
+            }
+            &.whitespace-bottom-#{$keyMap} {
+                  padding-bottom: $valueMap;
+            }
+        }
+    }
+
+    @each $keyMap, $valueMap in $sizes {
```

Comment:
This should really use separate variables for margin sizes, so we can change one, without risking change to the other.